Gems/Atom/RHI/Vulkan/Code/Source/RHI/StreamingImagePool.cpp

```diff
@@ -59,27 +87,68 @@ namespace AZ
                 }
             }
 
-            outHeapTiles = m_tileAllocator.Allocate(blockCount);
-            if (outHeapTiles.size() == 0)
+            AZStd::vector<VmaAllocation> allocations(blockCount);
+            VmaAllocationCreateInfo allocCreateInfo = GetVmaAllocationCreateInfo(RHI::HeapMemoryLevel::Device);
+
+            auto& device = static_cast<Device&>(GetDevice());
+            VkResult vkResult = vmaAllocateMemoryPages(
```

Comment:
is this function made for allocating sparse blocks and calling it pages? This needs comments as to what it is doing under the hood.